### ISSUE TITLE
permission denied osx. m1 mac

### ISSUE THREAD

**Issue by carlosgermanhurtado** [pystardust/ani-cli#1118]
**Metadata (please complete the following information)**
Version: [e.g. 2.0.2]
OS: osx ventura
Shell: zsh
Anime: ani-cli --dub

when running ani-cli from default terminal in ventura the resulting error message says "zsh permission denied "

**Comment by justchokingaround** [comment]
could you try running this commands to uninstall it, and install it please:
```sh
sudo rm $(which ani-cli)
brew install --cask iina
brew tap justchokingaround/homebrew-ani-cli-git
brew install justchokingaround/ani-cli-git/ani-cli-git
```

**Comment by Derisis13** [comment]
This probably happened because of the exec bit got unset. It has been fixed since in #1117

**Comment by port19x** [comment]
> This probably happened because of the exec bit got unset. It has been fixed since in #1117

Won't happen again thanks to #1124
